playground/AzureContainerApps/AzureContainerApps.AppHost/Program.cs

```diff
@@ -26,6 +26,11 @@
                    .RunAsEmulator(c => c.WithLifetime(ContainerLifetime.Persistent))
                    .AddBlobs("blobs");
 
+// Testing docker files
+
+builder.AddDockerfile("pythonapp", "AppWithDocker");
```

Comment:
would be good to update the manifest too

Reply:
Done